### DISCUSSION TITLE
Audio drops for no apparent reason

### DISCUSSION THREAD

**Discussion by manatactical** [apohl79/audiogridder#660] [Help]
Hi,

I've tried hard to use Audio Gridder. Sometimes it does work okay, sometimes it's just a mess.
Even an empty project with just one instrument (Audio Gridder), the sound drops (I see the drops on the network graph on the server).

My machines are fairly powerful (**Plugin-side**: iMac 10 cores 10900K, **server-side**: 18 cores 10980XE @ 4.4 GHz). Both machines are wired with a 2.5G ethernet cable, CAT 6.
Even with an absolute empty ASIO meter on my DAW, the red LED indicating glitches lights up.

This problem occurs on Cubase, Logic Pro, LUNA, Pro Tools Ultimate, and Ableton Live. 2 Blocks on Audio Gridder or higher, tested from 64 to 512 buffer size. 

Vienna Ensemble Pro is running perfectly smoothly. Any idea why Audio Girdder struggles that much?

Thanks for your help!

Best

**Comment by apohl79** [comment]
This seems to be a networking issue. What’s your exact network setup? Do you have a wifi interface in the same subnet as your wired interface?

**Reply by manatactical** [reply]
Hi,

Thanks for your reply.

Each machine is connected with its motherboard's built-in ethernet to the domestic network with its proper subnet (this one is used for everything but the audio).
They are also connected between them directly, with a dedicated PCI card and a different subnet (that's this one I'm using for audio transfer. This network is free of any perturbations. At least on the paper.

All the best

**Reply by apohl79** [reply]
You should see multiple servers in the AG plugin with their respective IPs if both connections are available. In this case, make sure you select the right IP.

If you just keep your audio connection active, do you still see issues? Maybe you can try each connection separately (while deactivating the other). I've seen issues you describe only on high latency connections like wifi.

**Reply by manatactical** [reply]
I force AG Server to bind a specific network. On the plugin machine, I always connect manually with the correct IP, just to be sure.
I will try furthers tests tomorrow. I will let you know about the audio connection active if the issues persist, or any useful information.

**Reply by manatactical** [reply]
The problem occurs even with just the audio connection active.

First, I've tried with different MTU sizes. Higher the MTU, worse was the experience.

After further investigations, Audio Gridder messes with the first thread of my mac. In some cases, Logic Pro reports a sample rate mismatch. I assume the one-thread CPU spike (which correlates with sound glitches and audio drops) indicates the machine trying to hold its shit with that sample mismatch.

https://imgur.com/YYz5gdR
